packages/dso-toolkit/src/components/highlight-box/highlight-box.mixins.scss

```diff
@@ -12,6 +12,17 @@
   margin-top: units.$u2;
   padding: units.$u2;
   margin-bottom: units.$vertical-spacing-large;
+
+  block-size: var(--highlight-box-block-size, unset);
+  min-block-size: var(--highlight-box-min-block-size, unset);
+
+  &[has-counter] {
```

Comment:
Suf eigenlijk dat we dit alleen ondersteunen voor de Web Component implementatie 🤷 

Maar laten we het vooral zo laten

Reply:
dat hele `:host-context` verhaal is ook alleen bedoeld voor webcomponents, toch?